service_config.go

```diff
@@ -229,15 +230,24 @@ type jsonName struct {
 	Method  *string
 }
 
-func (j jsonName) generatePath() (string, bool) {
+var errIgnoreName = errors.New("ignore name")
```

Comment:
Can you add more information to the error message?

Reply:
changed it to `errMissingService`